### DISCUSSION TITLE
can i use aiortc with django-channels. Is it alternative of webrtc for python. Any Good details reference for beginner

### DISCUSSION THREAD

**Discussion by jiwan-gharti** [aiortc/aiortc#720] [Q&A]
can i use aiortc with django-channels. Is it alternative of webrtc for python. Any Good details reference for beginner

**Comment by jiwan-gharti** [comment]
1. Is it solve the real time video conferencing with channel 
2.  Can we use AI in real time while live conferencing going on.
3.  Any Github link/blog that solves my doubts

**Reply by jlaine** [reply]
Your question is pretty vague:

- Can you use aiortc to do what? Sent video? Audio? Datachannels?
- How does Django come into the picture?
